test/elixir_bench/benchmarks/benchmarks_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule ElixirBench.BenchmarksTest do
+  use ElixirBench.DataCase
+  alias ElixirBench.{Benchmarks, Benchmarks.Job}
+  alias ElixirBench.Repos
+
+  describe "when correct params" do
```

Comment:
usually (well in my usage and iirc elixir core as well) `describe` isn't used for contexts or cases but rather methods so like `describe ".create_job/2" ... test "returns a new job given correct parametres"